sdk/identity/identity/src/credentials/managedIdentityCredential/msalMsiProvider.ts

```diff
@@ -182,7 +184,7 @@ export class MsalMsiProvider {
 
           if (!isAvailable) {
             throw new CredentialUnavailableError(
-              `ManagedIdentityCredential: The managed identity endpoint is not available.`,
+              `ManagedIdentityCredential: MSAL indicated that the IMDS endpoint is the source for managed identity, but the IMDS endpoint is not available.`,
```

Comment:
This could be misleading, since MSAL only can indicate that no other token source was definitively available based on environment configuration. IMDS is just the default fallback, including in local dev/test scenarios where this will always be the result from MSAL even though the endpoint is not there.
```suggestion
              `ManagedIdentityCredential: Attempted to use the IMDS endpoint, but it is not available.`,
```

Reply:
Great suggestions - applied both. Thank you!